### DISCUSSION TITLE
Get item "slots"

### DISCUSSION THREAD

**Discussion by tiddle** [TarkovTracker/tarkovdata#156] [Ideas]
Is there anyway to get the available slots for an item?

Like in the `parts` in the `item_presets.json`, but more specific. Eg, I want to know if a helmet is capable of housing a face shield or what is available to be modified on a gun.

**Comment by kokarn** [comment]
This is slowly being worked on with the tarkov-tools API but might be something that's suitable here as well.

There's some basic information about what BSG calls "linkedItems" in the tarkov-tools repo over here tho 
https://github.com/kokarn/tarkov-tools/blob/master/src/data/item-props.json

[Downloadable file](https://raw.githubusercontent.com/kokarn/tarkov-tools/master/src/data/item-props.json)
